runners/pom.xml

```diff
@@ -55,6 +55,13 @@
             <plugin>
               <groupId>org.apache.maven.plugins</groupId>
               <artifactId>maven-surefire-plugin</artifactId>
+              <dependencies>
+                <dependency>
+                  <groupId>org.apache.beam</groupId>
+                  <artifactId>beam-sdks-java-core</artifactId>
+                  <type>test-jar</type>
```

Comment:
The main artifact in beam-sdks-java-core that is produced uses the shade plugin. For any dependency that the test uses directly that is not a dependency of this module and also repackaged, bundled and stripped in beam-sdks-java-core would cause the test code to fail with class not found exceptions.